config/configgrpc/configgrpc.go

```diff
@@ -157,6 +157,9 @@ type GRPCServerSettings struct {
 	// Include propagates the incoming connection's metadata to downstream consumers.
 	// Experimental: *NOTE* this option is subject to change or removal in the future.
 	IncludeMetadata bool `mapstructure:"include_metadata,omitempty"`
+
+	// EnableReflection will add grpc reflection apis to the receiver
+	EnableReflection bool `mapstructure:"enable_reflection,omitempty"`
```

Comment:
What do you think about having a `reflection` section with an `enabled` boolean instead? In case we want to add any other reflection related settings in the future.

Reply:
sounds good, ill make the changes